FWCore/Utilities/interface/bit_cast.h

```diff
@@ -20,17 +20,38 @@
 
 // system include files
 #include <cstring>
+#include <type_traits>
 
-// user include files
+#if __cplusplus >= 202002L
+
+// in C++20 we can use std::bit_cast
+
+#include <bit>
```

Comment:
CPP reference says the class comes from <bitset>
https://en.cppreference.com/w/cpp/utility/bitset